### PR TITLE
[ID/EN] Add and update Disambiguation and group articles

### PR DESCRIPTION
**Pull request by hikikuman**
Add Indonesian translation for `Fail`, `NC`, `NM`, `Rank`, `Version`, and `osu!`. Update outdated Indonesian translation for `Disambiguation`, `Bancho`, `EZ`, `Easy`, and `Mod`. Also add `NC` and `NM` to the disambiguation list ([EN] and [ID] `Disambiguation`).

### CODE REVIEW

**Comment by hikikuman** [comment]
Re-add all commits. Also, add 2 new Indonesian translations for Combo and Offset.

File: wiki/Disambiguation/Bancho/id.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
----
-outdated: true
-outdated_since: 8043b3a603ef5a8bb7ab9f982fceed3825cb249f
----
+# Bancho (keambiguan)
```
Comment (SeverinoAji) [inline wiki/Disambiguation/Bancho/id.md:1]:
```suggestion
# Bancho (disambiguasi)
```
Kayanya buat kata "disambiguation" lebih cocok di translate ke "disambiguasi" daripada "keambiguan" , barusan nyari2 juga nemu di wikipedia indo (sbnrny krg cocok buat referensi :v) dan KBBI. Link : https://id.wikipedia.org/wiki/Disambiguasi_(disambiguasi)

File: wiki/Disambiguation/Combo/id.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Kombo (keambiguan)
```
Comment (SeverinoAji) [inline wiki/Disambiguation/Combo/id.md:1]:
```suggestion
# Kombo (disambiguasi)
```
masih sama kaya yg diatas tadi

File: wiki/Disambiguation/Combo/id.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+**Kombo** mempunyai beberapa arti:
+
+- [Pengali skor](/wiki/Glossary/Combo_(score_multiplier)) yang terlihat saat sedang bermain, atau hit object yang berhasil di-hit berturut-turut tanpa miss.
```
Comment (SeverinoAji) [inline wiki/Disambiguation/Combo/id.md:5]:
```suggestion
- [Pengali skor](/wiki/Glossary/Combo_(score_multiplier)) yang terlihat selama permainan berlangsung, atau hit object yang berhasil di-hit berturut-turut tanpa miss.
```

File: wiki/Disambiguation/EZ/id.md [hunk 4]
```diff
@@ -1,15 +1,10 @@
----
-outdated: true
-outdated_since: 8043b3a603ef5a8bb7ab9f982fceed3825cb249f
----
+# EZ (keambiguan)
```
Comment (SeverinoAji) [inline wiki/Disambiguation/EZ/id.md:1]:
```suggestion
# EZ (disambiguasi)
```

File: wiki/Disambiguation/Fail/id.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# Fail (keambiguan)
```
Comment (SeverinoAji) [inline wiki/Disambiguation/Fail/id.md:1]:
```suggestion
# Fail (disambiguasi)
```

File: wiki/Disambiguation/Mod/id.md [hunk 6]
```diff
@@ -1,11 +1,6 @@
----
-outdated: true
-outdated_since: 8043b3a603ef5a8bb7ab9f982fceed3825cb249f
----
+# Mod (keambiguan)
```
Comment (SeverinoAji) [inline wiki/Disambiguation/Mod/id.md:1]:
```suggestion
# Mod (disambiguasi)
```

File: wiki/Disambiguation/Mod/id.md [hunk 7]
```diff
@@ -1,11 +1,6 @@
...
 
-# Mod
-
-**Mod** mungkin juga merujuk pada:
+**Mod** mempunyai beberapa arti:
 
 - Seorang moderator, seperti anggota [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) atau [GMT](/wiki/People/The_Team/Global_Moderation_Team).
 - Seseorang yang melihat sebuah beatmap dan memberikan umpan balik.
```
Comment (SeverinoAji) [inline wiki/Disambiguation/Mod/id.md:6]:
```suggestion
- Seseorang yang memeriksa sebuah beatmap dan memberikan umpan balik.
```

File: wiki/Disambiguation/Offset/id.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# Offset (keambiguan)
```
Comment (SeverinoAji) [inline wiki/Disambiguation/Offset/id.md:1]:
```suggestion
# Offset (disambiguasi)
```

File: wiki/Disambiguation/Version/id.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+# Versi (keambiguan)
```
Comment (SeverinoAji) [inline wiki/Disambiguation/Version/id.md:1]:
```suggestion
# Versi (disambiguasi)
```

File: wiki/Disambiguation/id.md [hunk 10]
```diff
@@ -1,15 +1,17 @@
----
-outdated: true
-outdated_since: 8043b3a603ef5a8bb7ab9f982fceed3825cb249f
----
-
 # Keambiguan
```
Comment (SeverinoAji) [inline wiki/Disambiguation/id.md:1]:
```suggestion
# Disambiguasi
```

File: wiki/Disambiguation/osu!/id.md [hunk 11]
```diff
@@ -0,0 +1,7 @@
+# osu! (keambiguan)
```
Comment (SeverinoAji) [inline wiki/Disambiguation/osu!/id.md:1]:
```suggestion
# osu! (disambiguasi)
```

File: wiki/Disambiguation/osu!/id.md [hunk 12]
```diff
@@ -0,0 +1,7 @@
...
+
+**osu!** mempunyai beberapa arti:
+
+- Gim dan komunitasnya
```
Comment (SeverinoAji) [inline wiki/Disambiguation/osu!/id.md:5]:
```suggestion
- Game dan komunitasnya
```
mungkin yang game bisa tetap di buat game (?) atau emang harus di translate ke Gim?

File: wiki/Disambiguation/Bancho/id.md (hunk 1 above)
Comment (hikikuman) [reply wiki/Disambiguation/Bancho/id.md:1]: umm, en nya `disambiguation`, id nya `disambiguasi`, bener juga ya... setuju kalo gitu

File: wiki/Disambiguation/osu!/id.md (hunk 12 above)
Comment (hikikuman) [reply wiki/Disambiguation/osu!/id.md:5]: pernah didiskusiin ini, pakenya `gim, bukan `game`.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**